.travis.yml

```diff
@@ -8,369 +9,407 @@ cache:
   bundler: true
   pip: true
   directories:
-  - $HOME/.cache/pre-commit
-  - $HOME/.pre-commit
-  - $HOME/.rvm
-  - $HOME/virtualenv/python$(python -c 'import platform; print(platform.python_version())')
-  - $HOME/Library/Caches/Homebrew
+    - $HOME/.cache/pre-commit
```

Comment:
I like previous style more, I used it on purpose. It helps to maintain shorter lines even for nested struts

Reply:
This is not a matter of opinion, the changes were requested by yamllint. I can rollback no problem.

Reply:
Lets go with yamllint